### DISCUSSION TITLE
Error adding thread to existing Assistant.

### DISCUSSION THREAD

**Discussion by joycpa** [RageAgainstThePixel/OpenAI-DotNet#363] [Q&A] [answered]
Am trying to add a thread to and an Assistant that was started from the OpenAI playground. It includes a file that I want to include in the thread. I am using the OpenAI-DotNet library.
HttpRequestException: StreamEventsAsync Failed! HTTP status code: BadRequest | Response body: { "error": { "message": "Invalid 'tools[1].file_search.max_num_results': integer below minimum value. Expected a value >= 1, but got 0 instead.", "type": "invalid_request_error", "param": "tools[1].file_search.max_num_results", "code": "integer_below_min_value" } }

**Comment by StephenHodgson** [comment]
@joycpa did you create your asst in the playground or with the API?

**Comment by joycpa** [comment]
In the Playground. I am trying to add the thread in the API. I am trying to attach a file to the thread.

 

Thanks

Myron

**Reply by StephenHodgson** [reply]
Thanks for clarifying. Yes it is [a known issue](https://github.com/RageAgainstThePixel/OpenAI-DotNet/issues/364) I plan to fix in next release. Thanks for reporting!

**Comment by joycpa** [comment]
Thanks for the update, any idea of when the next release will be?

**Reply by StephenHodgson** [reply]
Probably over the weekend.

**Reply by StephenHodgson** [reply, marked as answer]
@joycpa should be fixed in https://github.com/RageAgainstThePixel/OpenAI-DotNet/releases/tag/v8.2.4

**Reply by joycpa** [reply]
Thanks for your help. I tried it and it works perfect.
